### DISCUSSION TITLE
Version Upgrade Category Taxonomies Issue

### DISCUSSION THREAD

**Discussion by ashishnjoshi** [statamic/cms#4985] [Troubleshooting]
I need help on below point-

We had created Category Taxonomies (Taxonomies>>Category) and found that in previous Statamic version(3.1.24) all the categories were shown even if there is no post under the category and after the Statamic version update (3.2.10) it's automatically filtering category listing and show only those categories which has a post available.
In older version(3.1.24) there was all categories listed and after Statamic version we don't have any changes in code level but just after version upgrade all categories listing are not showing.
So, please let us know how to fix this issue.
Sample Code : 

 **{{ categories }}

            {{ partial:posts/categories/card }}

 {{ /categories }}**

------------**card.antlers.html file code** 

    **<a href="{{ url }}">
        <p class="pt-6 text-2xl text-white font-bold px-5">{{ title }}</p>
      </a>
      {{ if description }}
           <div class="pt-4 text-lg text-white font-light px-5 text-opacity-75">{{ description }}</div>
        {{ /if }}**

**Comment by duncanmcclean** [comment]
Conversation on Discord for reference (which includes the issue details): https://discord.com/channels/489818810157891584/489864118719873024/925602619739734057

**Comment by jasonvarga** [comment]
This changed in 3.1.25. https://github.com/statamic/cms/pull/3870

Unfortunately you were relying on a buggy behavior. The fix is how it's always been documented.

You should be able to add `min_count="0"` to show all the terms even if they have no entries.
